port/linux/readme.md

```diff
@@ -70,4 +70,12 @@ Option and default setting in the Linux Makefile
   
 - WKCORE
 
-  enable discovery through IETF /.well-known/core on IETFs multicast ALL COAP NODES
\ No newline at end of file
+  enable discovery through IETF /.well-known/core on IETFs multicast ALL COAP NODES
+
+- PUSH 0
```

Comment:
Isn't this inverted? 

PUSH 1 -> enabled, PUSH 0 -> disabled

Reply:
oos, that's right
fixed